pkg/bootstrap/bootstrap.go

```diff
@@ -221,7 +233,14 @@ func (c *BootstrapClient) applyManifests(ctx context.Context, objs []unstructure
 
 		if err = retry.RetryOnConflict(retry.DefaultRetry, func() error {
 			_, err = c.client.Resource(mapping.Resource).Namespace(obj.GetNamespace()).
-				Apply(ctx, obj.GetName(), &obj, metav1.ApplyOptions{Force: true, FieldManager: "glasskube"})
+				Apply(
+					ctx, obj.GetName(),
+					&obj,
+					metav1.ApplyOptions{
+						Force:        true,
+						FieldManager: "glasskube",
+						DryRun:       []string{metav1.DryRunAll},
```

Comment:
The `DryRun` property should only be set if dry-run is enabled via options

Reply:
```go
var dryrun []string
		if options.DryRun {
			dryrun = []string{metav1.DryRunAll}
		}

		if err = retry.RetryOnConflict(retry.DefaultRetry, func() error {
			_, err = c.client.Resource(mapping.Resource).Namespace(obj.GetNamespace()).
				Apply(
					ctx, obj.GetName(),
					&obj,
					metav1.ApplyOptions{
						Force:        true,
						FieldManager: "glasskube",
						DryRun:       dryrun,
					})
			return err
		}); err != nil {
			return err
		}
		```
		
		Something like this would do?

Reply:
yes something like that